lib/eval.js

```diff
@@ -143,7 +143,7 @@ const evalLiteral = node => {
   assert.strictEqual(node.type, NODE_LITERAL);
 
   const { value } = node;
-  const typeNames = [TYPE_NUMBER, TYPE_BOOLEAN, TYPE_STRING];
+  const typeNames = [TYPE_NUMBER, TYPE_BOOLEAN, TYPE_STRING, TYPE_OBJECT];
```

Comment:
Please remove `TYPE_OBJECT` from `typeNames` since it would be an error if a node of type `NODE_LITERAL` has an object as value.